### PR TITLE
Add brief user doc with screenshots

### PR DESCRIPTION
**Pull request by julien-nc**
* brief user doc
* add links to docs in README and info.xml
* add link to https://docs.nextcloud.com/server/latest/admin_manual/ai/index.html in docs main page

### CODE REVIEW

File: docs/user/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+![top right icon](../img/notifications.png)
+
+## History
+
+On the bottom left corner of the assistant, there is a button display the task history.
+This task history is filtered to only show tasks of the selected task type.
+You can then relaunch previous tasks, delete them or just cancel them if they were scheduled.
+
+![top right icon](../img/history.png)
```
Comment (kyteinsky) [inline docs/user/README.md:38]: the alt of the images needs an update. lgtm otherwise :tada:

**Comment by DaphneMuller** [comment]
It looks good to me, please also document how to access the smart pickers

**Review by DaphneMuller — APPROVED** [review]

File: docs/user/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+You task will run immediately if possible or be scheduled for later execution.
+
+![getting results](../img/getting.results.png)
+
+![scheduled](../img/scheduled.png)
+
+## Notifications
+
+If it is scheduled, you will receive a notification when it has finished.
```
Comment (marcelklehr) [inline docs/user/README.md:27]:
```suggestion
If the task was scheduled, you will receive a notification when it has finished.
```

File: docs/user/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+You can display the task results by clicking the "View results" button in the notification.
+
+![notifications](../img/notifications.png)
+
+## History
+
+On the bottom left corner of the assistant, there is a button display the task history.
```
Comment (marcelklehr) [inline docs/user/README.md:34]:
```suggestion
On the bottom left corner of the assistant, there is a button to display the task history.
```

File: docs/user/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+This task history is filtered to only show tasks of the selected task type.
```
Comment (marcelklehr) [inline docs/user/README.md:35]:
```suggestion
This task history is filtered to only show tasks of the currently selected task type.
```

**Review by marcelklehr — APPROVED** [review]

File: docs/user/README.md (hunk 1 above)
Comment (julien-nc) [reply docs/user/README.md]: :see_no_evil:

**Comment by julien-nc** [comment]
Thank you all :blue_heart:

**Pull request merged**